### DISCUSSION TITLE
Specific version of Docker?

### DISCUSSION THREAD

**Discussion by dlover25** [teslamate-org/teslamate#2906] [General]
Does TeslaMate have specific versions of Docker that are supported? I initially built my TeslaMate instance on Ubuntu 20.04 with Docker 19.03.8. Working great. long time ago, I accidentally let ubuntu upgrade Docker to something newer that completely broke TeslaMate. I reverted back to 19.03.8 and it worked again. So, I put an Apt Hold on that version. I'm moving this to a new virtual machine host, and using ubuntu 22.04, with default docker. 20.10.14. Doesn't work. So, wondering what version I should go to. Obviously, I could go back to 19.03.8, but I'd love to know what I can go to. The other containers seem to be ok. But Teslamate crashes. 

![image](https://user-images.githubusercontent.com/55103311/191306559-b8f51f29-e7df-462a-aff6-e91f86cb5bf9.png)

**Comment by DrMichael** [comment]
Running it under Ubuntu 22.04.1 with Docker version 20.10.18, build b40c2f6, with no problem.
And under Raspbian GNU/Linux 11 with Docker version 20.10.5+dfsg1, build 55c4c88.

**Reply by dlover25** [reply]
I must be installing something wrong. Can I ask how you installed docker/docker-compose on your ubuntu 22.04.1?
